documentation/api/README.md

```diff
@@ -24,6 +24,6 @@ The following are the root routes on the HTTP API surface.
 | [`/info`](info.md) | Gets info about `dotnet monitor`. | 6.0 |
 | [`/operations`](operations.md) | Gets egress operation status or cancels operations. | 6.0 |
 | [`/collectionrules`](collectionrules.md) | Gets the current state of collection rules. | 6.3 |
-|  [`/parameters`](parameters.md) | Captures parameters for one or more methods each time they are called. | 8.0 RC 1 |
+| [`/parameters`](parameters.md) | Captures parameters for one or more methods each time they are called. | 8.0 RC 1 (breaking change in 9.0 Preview 3) |
```

Comment:
Open question for more of the dotnet-monitor folks, do we want to call out breaking changes for experimental features? I'm personally in favor of not doing so since experimental features provide no support guarantees are subject to change until they are no longer experimental.

Reply:
@clguiman We discussed this offline and we decided on not mentioning breaking changes in our documentation for experimental features (i.e. omit `breaking change in 9.0 Preview 3` here, and don't make mention of how things used to work before this change)